site/source/user-manual/flashing/flashing-pi.rst

```diff
@@ -21,53 +21,34 @@ Select your OS to continue:
 
     .. group-tab:: Linux
 
-        #. Open a terminal in the folder you downloaded to and extract with:
+        (Optional, but recommended) Verify the checksum against the one listed on GitHub (SHA256):
         
             .. code-block::
             
-                tar -xzvf startos-0.3.4.2-bbd66e9-20230519_raspberrypi.img.gz
-
-        #. (Optional, but recommended) Verify the checksum against the one listed on GitHub (SHA256):
-        
-            .. code-block::
-            
-                sha256sum startos-0.3.4.2-bbd66e9-20230519_raspberrypi.img
-
-            or (BLAKE3):
```

Comment:
Removed blake3 sum instructions because unless they have cargo installed and have already done `cargo install b3sum`, b3sum will not be available to them.  SHA256 is fine, I think, and what the windows and mac guides use.  If they're sophisticated enough to know they want a blake3 sum, this guide won't stop them.

Reply:
I'm not sure what you mean.  It will definitely stop them from knowing what the b3sum of the file we published was

Reply:
They definitely can still check the blake3 sum, as it's still on the github page.  There'll just be no instructions for verifying the b3sum.  I'm not against blake3 by any means - I actually like it, it seems faster and better than sha256.  I just think it's needlessly verbose and complicates the instructions to offer an alternative that is available on every system - at best - and will eventually cause people to ask "How come I don't have b3sum, and where are the instructions for installing it?", especially considering what it takes to actually install it -- rust has to be installed and they need a functional cargo environment, and then they need to run `cargo install b3sum`.  For anyone determined to check the b3sum instead of the sha256sum, still nothing will stop them now.  Point being that simply knowing you want to use blake3 instead of sha256 kinda implies you know what you're doing and don't need instructions.

Reply:
My mistake, I thought this was the startos repo... makes sense to me